client/wfnews-war/src/main/angular/src/app/components/current-alert-component/current-alert-component.component.html

```diff
@@ -52,7 +52,66 @@
 
     <div class="notice-list">
         <div *ngFor="let evac of evacOrders" class="notice-item">
-            {{evac.eventName}}
+            <div class="item-header">{{evac.eventName}}</div>
+            <span class="label"><wf-icon class="icon" iconName="error" iconSize="small"></wf-icon><span style="padding-left: 10px; margin-top: 5px;">Evacuation Orders and Alerts</span></span>
+            <div class="item-row">
+                <span class="item-title">Last Updated: </span>
+                <span class="item-content">{{convertToDateWithDayOfWeek(evac.dateModified)}}</span>
+            </div>
+            <div class="item-row">
+                <span class="item-title">Agency: </span>
+                <span class="item-content">{{evac.issuingAgency}}</span>
+            </div>
+            <div class="item-row">
+                <span class="item-title">Status: </span>
+                <span class="item-content">{{evac.orderAlertStatus}}</span>
+            </div>
+            <div class="item-row">
+                <span class="item-title">Distance from My Location: </span>
+                <span class="item-content">{{calculateDistance(evac.centroid.y, evac.centroid.x)}} km</span>
+            </div>
+        </div>
+
+        <div *ngFor="let ban of bansAndProhibitions" class="notice-item">
+            <div class="item-header" style="background: url('../../../assets/images/external-link.svg') center right no-repeat; background-size: 18px;" (click)="openBcGovGuide()">{{ban.type}}</div>
+            <span class="label"><wf-icon class="icon" iconName="block" iconSize="small"></wf-icon><span style="padding-left: 10px; margin-top: 5px;">Bans and Prohibitions</span></span>
+            <div class="item-row">
+                <span class="item-title">Last Updated: </span>
+                <span class="item-content">{{convertToDateWithDayOfWeek(ban.accessStatusEffectiveDate)}}</span>
+            </div>
+            <div class="item-row">
+                <span class="item-title">Fire Centre: </span>
+                <span class="item-content">{{ban.fireCentre}}</span>
+            </div>
+            <div class="item-row">
+                <span class="item-title">Restrictions: </span>
+                <span class="item-content">{{convertRestriction(ban.accessProhibitionDescription)}}</span>
+            </div>
+            <div class="item-row">
+                <span class="item-title">Distance from My Location: </span>
+                <span class="item-content">{{calculateDistance(ban.centroid.y, ban.centroid.x)}} KM</span>
```

Comment:
minor niggle, but I think we used lowercase km elsewhere. We can double check for consistency later